client/src/App.js

```diff
@@ -1,26 +1,60 @@
 import React,{useState} from 'react';
-import styled from 'styled-components';
+import styled,{createGlobalStyle} from 'styled-components';
 import Login from './compontents/login/Login';
 import Register from './compontents/register/Register';
-import './App.css';
+
+const GlobalStyles=createGlobalStyle`
+body{
+  font-family: 'Arimo', sans-serif;
+}
+`
+
+
+const AppWrap=styled.div`
+position:absolute;
+top:0px;
+background:rgb(44, 8, 82);
+width:100%;
+height:45%;
+`
 
 const H1=styled.h1`
-color:rgb(44, 8, 82);
+color:white;
+margin-top:50px;
 margin-left:10%;
 `
+const FormWrapper=styled.div`
+display:flex;
+flex-wrap:wrap;
+margin:80px auto auto auto;
+width:70%;
+@media (max-width:480px){
+  width:90%;
+}
+`
+
 
 function App() {
-  const [user,setUser]=useState('');
+  const [name,setName]=useState('');
   const [status,setStatus]=useState(false);
+  const [newUser,setNewUser]=useState(false);
+
+  function registerForm(){
+    setNewUser(true)
+  }
+
+  function loginForm(){
+    setNewUser(false)
+  }
```

Comment:
Could we compress this into one? Maybe something like `newUser ? setNewUser(false) : setNewUser(true)`

Reply:
yes, I can do that.